.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,57 @@
+name: Bug Report
+description: File a bug report
+labels:
+  - 'status: triage'
+  - 'type: bug'
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to submit a bug report!
+  - type: markdown
+    attributes:
+      value: >-
+        Please note that support for compiling the mod is not currently provided.
+
+        For help with Galacticraft 4 and below, please use [TeamGalacticraft/Galacticraft-Legacy](https://github.com/TeamGalacticraft/Galacticraft-Legacy).
+
+        For general Galacticraft help and support visit the [Galacticraft Central Discord](http://discord.galacticraftcentral.com/)
+  - type: dropdown
+    id: modloader
+    attributes:
+      label: Mod loader
```

Comment:
Only thing that I think could be changed is capitalizing Loader

Reply:
Fixed in f4b14d1eb90a6eaf80196f1c881fb0f31a21a736